website/docs/d/google_compute_regions.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "google"
+page_title: "Google: google_compute_regions"
+sidebar_current: "docs-google-datasource-compute-regions"
+description: |-
+  Provides a list of available Google Compute regions
+---
+
+# google\_compute\_regions
+
+Provides access to available Google Compute regions for a given project.
+See more about [regions and regions](https://cloud.google.com/compute/docs/regions-regions/regions-regions) in the upstream docs.
```

Comment:
This link is broken. And `regions and regions` is a bit redundant ;)

I believe you wanted to link to regions and zones?
https://cloud.google.com/compute/docs/regions-zones/

Reply:
yes sorry.
changed